src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,118 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMax.SoftLimitDirection;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import static frc.robot.Constants.CAN;
+import static frc.robot.Constants.ClimbSettings;
+
+public class Climber {
+    // PIDSlot used
+    int slot = 0;
+
+    // raise/lower controllers
+    private CANSparkMax left_extender= new CANSparkMax(CAN.CMB_LEFT_Extend, MotorType.kBrushless);
+    private CANSparkMax right_extender = new CANSparkMax(CAN.CMB_RIGHT_Extend, MotorType.kBrushless);
+
+    // rotation arm controller (outer arms rotate)
+    // private CANSparkMax l_rotator = new CANSparkMax(CAN.CMB_L_Rotate, MotorType.kBrushless);
+    // private CANSparkMax r_rotator = new CANSparkMax(CAN.CMB_R_Rotate, MotorType.kBrushless);
+
+    public Climber() {
+        // TODO - raise/lower are actions ie commands. In the sub-sys we are defining
+        // devices
+        // and behaviors. Left/Right Inner/Outer arms left-rotator, right-rotator
+        // gives a better idea of what
+
+        // TODO Set the control type and PID settings - most likely position control
+
+        // Copy the PID settings down to the SparkMax hardware 
+        // Rotation PIDS
+        // ClimbSettings.armPID.copyTo(l_rotator.getPIDController(), slot);
+        // ClimbSettings.armPID.copyTo(r_rotator.getPIDController(), slot);
+        // // Arm extension PIDS
+        ClimbSettings.innerPID.copyTo(left_extender.getPIDController(), slot);
+        ClimbSettings.innerPID.copyTo(right_extender.getPIDController(), slot);
+
+
+    }
+
+
+    public boolean readyToClimb() {
+        // returns if robot is in the right position, and all motors are in place
+        return false;
+    }
+
+    public String rung() {
+        return "blank";
+    }
+
+    public boolean readyToTraverse() {
+        return false;
+    }
+    // @param inches from extender current position
```

Comment:
We need to reconsider if it is smarter to base our 0 off the current position or fully retracted. I am starting to think 0 being set at fully retracted is the best bet. All values should be positive in this case, need to enforce that.